### ISSUE TITLE
wmt数据下载

### ISSUE THREAD

**Issue by 0-KaiKai-0** [huawei-noah/Pretrained-Language-Model#226]
请问论文Universal Conditional Masked Language Pre-training for Neural Machine Translation中所描述的数据集size是指什么，以及能否提供论文中所使用的数据下载源。
![image](https://user-images.githubusercontent.com/60919370/204527286-044f6cde-2341-4e30-8e7a-d6359951b174.png)

**Comment by jingmu123** [comment]
您好，这个数据是从WMT官网下载并清洗之后的数据，size是指用于训练的数据规模，和mBART论文中一致；
